biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -75,12 +81,41 @@ subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort )
     logical, parameter :: test_zero_mortality = .false. ! Developer test which
                                                         ! may help to debug carbon imbalances
                                                         ! and the like
+     
+   ! Size Dependent Senescence
+    ! rate (r) and inflection point (ip) define the increase in mortality rate with dbh
+    mort_r_size_senescence = EDPftvarcon_inst%mort_r_size_senescence(cohort_in%pft)
+    mort_ip_size_senescence = EDPftvarcon_inst%mort_ip_size_senescence(cohort_in%pft)
+
```

Comment:
I'm worried that a user will set mort_r_size_senescence to 0 and think that will turn off size dependent senescence.  I think that value generates an smort of 0.5.  Will check how this is read in.

Reply:
Lets add some metadata to the parameter value entry that explains _ turns this off, and what valid ranges are.
https://github.com/NGEET/fates/blob/master/parameter_files/fates_params_default.cdl#L322

There is also this subroutine here, a good place to check if fates parameters are in acceptable ranges (if for instance 0 is an unexpected/invalid, you can have fates gracefully complain and fail):
https://github.com/NGEET/fates/blob/d8e7935542222ca2333d7007c5b394a461538331/main/EDPftvarcon.F90#L2071

Reply:
Sounds good. I'll add some metadata to the parameter value entry and some checks in FatesCheckParams.